### DISCUSSION TITLE
[Feature Request] Disable Find Functionality

### DISCUSSION THREAD

**Discussion by kycutler** [microsoft/monaco-editor#3784] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

I want to implement custom search functionality that works across several Monaco editors at once. I want this functionality to replace the Monaco built-in find & replace functionality entirely, but I don't see any way to disable the built-in Find functionality.

I would like an approach that is more bulletproof than simply overwriting the existing keyboard shortcuts.

**Comment by stefandobre** [comment]
If you go the [webpack plugin](https://github.com/microsoft/monaco-editor/tree/main/samples/browser-esm-webpack-monaco-plugin) route and build the library youtself, you can choose which features to exclude. I believe Find and Replace is one of them. Here's [an example](https://github.com/microsoft/monaco-editor/issues/2730) I created a while back for a slightly different usecase.

**Comment by goliney** [comment]
Is it possible to **optionally** disable Find functionality? I have places in my app where I'd like to preserve Find.

**Comment by matyax** [comment]
I achieved it with:

```
monaco.editor.addKeybindingRule({
  keybinding: monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyF,
  command: null
});
```

Version 0.34.1.
